modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/utils/SSH.scala

```diff
@@ -104,16 +102,15 @@ object SSH {
       .traverse(SecretStore[F].getValue)
       .map(key => Identity(tunnelConfig.bastion.passphrase.map(_.getBytes), key.map(_.getBytes)))
 
-  def connectedSession[F[_]: Sync: ContextShift](
-    blocker: Blocker,
+  def connectedSession[F[_]: Sync](
     config: TunnelConfig,
     identity: Identity
   ): Resource[F, Session] =
     for {
       _ <- Resource.eval(Logger[F].info("Creating new SSH session"))
       session <- Resource.make(createSession(config, identity))(s => Sync[F].delay(s.disconnect()))
```

Comment:
It's this way because when closing the resource it is important to tear down the port forwarding before tearing down the connection.

Unfortunately I cannot remember the error message which led me to do this.  Previous releases had big problems with SSH tunnels getting stuck, whereas the current release is quite stable.  I would prefer to leave it like this as it works.

Reply:
I wasn't aware of the history of this. Of cause leave it working, but add a comment.